cms/static/sass/_variables.scss

```diff
@@ -15,8 +16,12 @@
 
 $baseline: 20px;
 
+// +Paths
+// ====================
+$static-path: '..' !default;
+
 // +Grid
-// ==================== 
+// ====================
```

Comment:
Nit: Avoid mixing unrelated whitespace changes into such a large
changeset as this; the PR is already massive. While "nice" to do some
clean up, it not only makes this initial review more difficult, but it
makes it harder to digest the changeset in the future.

The more noise, the easier it is to lose the signal.